core/controllers/story_editor_test.py

```diff
@@ -319,11 +319,14 @@ def test_delete_can_not_access_story_handler_with_invalid_story_id(self):
 
     def test_delete_can_not_access_story_handler_with_invalid_topic_id(self):
         self.login(self.ADMIN_EMAIL)
-        topic_services.delete_topic(self.admin_id, self.topic_id)
+        new_story_id = story_services.get_new_story_id()
+        self.save_new_story(
```

Comment:
The test does expect an error here. 
If you mean the `save_new_story` function. It is a function used only for testing and just saves the story without checking the topic_id, so an error wouldn't happen on saving.

Reply:
Yeah, I mean save_new_story. But shouldn't save_new_story() be calling the regular methods which save a story, and those methods should throw an error?

I.e. if it is impossible to save a story without a topic ID in the real app, then we should make that be the case too in the test app.

Reply:
In the real app, this is called from topic_editor, so the decorator we have there would throw an error if an invalid topic_id is provided. There are no further validations, since everytime we validate, we'll have to have a database query to fetch the topic, so just the one topic validation at the controller layer is there. 
Whereas in the test app, we might need to save with invalid topic for testing.

Reply:
Oh, I see -- hm, OK. In that case, in the self.save_new_story() function docstring, could you please add the information "callers are responsible for ensuring that the topic ID provided is valid"?

Reply:
Done